Cabal/Distribution/Types/MungedPackageName.hs

```diff
@@ -137,7 +137,7 @@ zdashcode s = go s (Nothing :: Maybe Int) []
 
 parseZDashCode :: CabalParsing m => m [String]
 parseZDashCode = do
-    ns <- P.sepBy1 (some (P.satisfy (/= '-'))) (P.char '-')
+    ns <- toList <$> P.sepBy1 (some (P.satisfy (/= '-'))) (P.char '-')
```

Comment:
should there be `P.sepByNonEmpty` or no `toList`?

Reply:
Yep, the `toList` here is redundant.